packages/admin-ui-docs/content/form/code.mdx

```diff
@@ -49,31 +49,159 @@ To understand how this library works you need first to understand some concepts.
 
 </Section>
 
-<Section>
+<Section hasExplanation>
 
 ### Name
 
 This property specifies the name of the input, and it is used to subscribe to it. You can pass this prop to every component in this library.
 
 </Section>
 
-<Section>
+<Section hasExplanation>
 
 ### Subscriptions
 
-Is the ability to watch individual inputs and formState updates without re-rendering the entire form. There are two ways of subscribing to an input element: by using the `Watch` component or the `useWatch` hook.
+The ability to watch individual inputs and formState updates without re-rendering the entire form. There are two ways of subscribing to an input element: using the `Watch` component or the `useWatch` hook. They can be used as a replacement for the `onChange` callback.
 
 </Section>
 
 <Section>
 
+#### Watch
```

Comment:
I think it's better to add those examples to the `overview` instead of `code` section

Reply:
The watch and useWatch section is very similar to what I've added: [watch](https://admin-ui-docs-8b3zvb9ik-admin-team.vercel.app/guidelines/components/form#watch) and [useWatch](https://admin-ui-docs-8b3zvb9ik-admin-team.vercel.app/guidelines/components/form#usewatch)